ApollosProject/apollos-embeds
# Web qa

### Basecamp Scope:

[(Web) Search Embeds has a z-index issue with the church logo](https://3.basecamp.com/3926363/buckets/27088350/todos/6423916882/)

[(Web) Search Embed put Pages above Content on results](https://3.basecamp.com/3926363/buckets/27088350/todos/6423908592/)

[(Web) Add slide up animation to the modal on enter/exit](https://3.basecamp.com/3926363/buckets/27088350/todos/6423864906/)

## What was done:

- webflow cedar creek logo at `z-index 999`, so sent `z-index` for search to 1000
- restructured how collections were being rendered, switched `pages` and `content` positions
- added a sliding up and down animation to the modal

### Note: 
~~had to fiddle around to get Sliding down animation to work how I wanted it to, might be some issues relating to refreshes and url changes~~. 
Adding animations actually exposed these issues, rather than cause them.

**Relevant issue by adding animations tracked here:**

[(Web) View all showing both modal and web view](https://3.basecamp.com/3926363/buckets/27088350/todos/6438871997/)

[(Web) Two instances of modal/feed on refresh when searchParam is not null](https://3.basecamp.com/3926363/buckets/27088350/todos/6442722721/)

## How to test:

- open search, see that pages is above content
- Click around to see modal animations

<img width="1044" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/68402088/850af539-7ae7-497e-8327-a355fea85b96">

https://github.com/ApollosProject/apollos-embeds/assets/68402088/2530762a-e8e8-4324-ad30-6cb53c67ef35
